### PR TITLE
docs: Migrate docs pm to npm from yarn

### PR DESCRIPTION
**Pull request by kevintyj**
## What changed / motivation ?
I noticed that the project was mainly using npm, however, the docs app (website) was using yarn. I have edited the package.json to migrate yarn commands to npm and updated README to reflect the changes

## Linked PR/Issues
None

## Additional Context

- I would propose migrating to [pnpm](https://pnpm.io/pnpm-vs-npm)
- Also added documentation about building stylex packages to README
- **Important!**: It seems that stylelint fails due to a missing package `stylelint-copyright` and `lint & ci` commands seems to fail. Was not sure if this was expected behavior and if the linter is only ran on ci.
- Testing was done to make sure current behavior does not change
- Added `**/build/**` to stylelint (was not sure if this is a desired behavior)

## Pre-flight Checklist
- [x] I have read the [contributing guidelines](https://github.com/facebook/stylex/blob/main/.github/CONTRIBUTING.md)
- [x] I have signed the contributing [agreement](https://code.facebook.com/cla)
- [ ] I have written unit tests where necessary (If it is a feature commit)
- [x] Performed a self-review of my code
- [x] Made sure PR title follows the conventional commit specification

### CODE REVIEW

File: apps/docs/README.md [hunk 1]
```diff
@@ -6,24 +6,28 @@ This website is built using [Docusaurus 2](https://docusaurus.io/), a modern sta
 
 - stylex actual package, and then the @stylexjs/babel-plugin plugin in node_modules
 
+```bash
+$ npm run build -w @stylexjs/stylex -w @stylexjs/shared -w @stylexjs/eslint-plugin -w @stylexjs/babel-plugin
```
Comment (necolas) [inline apps/docs/README.md:10]: `npm run build --workspaces` is all that's needed

**Review by necolas — APPROVED** [review]
LGTM

File: apps/docs/README.md (hunk 1 above)
Comment (kevintyj) [reply apps/docs/README.md:10]: That builds all the packages/apps in the workspace, the documentation above only mentions the stylex packages and eslint plugin, so I only added builds that are required for docs to run (minimizes build time for first time contributors looking to only contribute to docs as size of monorepo increases). I could change it to build the entire workspace if that is the desired behavior 😀

**Review by nmn — APPROVED** [review]
Nice!

**Pull request merged**